RCAIDE/Framework/Mission/Sequential_Segments.py

```diff
@@ -0,0 +1,123 @@
+# RCAIDE/Framework/Mission/Sequential_Segments.py
+# (c) Copyright 2023 Aerospace Research Community LLC
+# 
+# Created:  Jun 2024, M. Clarke 
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  IMPORT
+# ----------------------------------------------------------------------------------------------------------------------  
+# RCAIDE imports   
+import RCAIDE
+from RCAIDE.Library.Mission.Common.Segments    import  sequential_segments
+from RCAIDE.Library.Mission.Common.Pre_Process import  aerodynamics,stability,energy,set_residuals_and_unknowns
+from RCAIDE.Framework.Core                     import Container as ContainerBase
+from RCAIDE.Framework.Analyses                 import Process 
+from . import Segments
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Sequential Segments 
+# ----------------------------------------------------------------------------------------------------------------------   
+class Sequential_Segments(Segments.Segment.Container):
+    """ Solves each segment sequentially, one at a time
+    """
+    
+    def __defaults__(self):
+        """This sets the default values.
+    
+            Assumptions:
+            None
+    
+            Source:
+            None 
+        """          
+
+        self.tag = 'mission'
+        
+        #   Initialize   
+        self.process.initialize                                = Process() 
+        self.process.initialize.aero                           = aerodynamics
+        self.process.initialize.stability                      = stability
+        self.process.initialize.energy                         = energy
+        self.process.initialize.set_residuals_and_unknowns     = set_residuals_and_unknowns
+ 
+        #   Converge 
+        self.process.converge    = sequential_segments
+         
+        #   Iterate     
+        del self.process.iterate  
+
+        return  
+
+    def pre_process(self):
+        """ This executes a pre-processing step
+    
+            Assumptions:
+                None
+    
+            Source:
+                None
+    
+            Args:
+                self (dict): RCAIDE data structure of containing process [-]
+    
+            Returns:
+                self (dict): RCAIDE data structure of containing process [-]
+        """   
+        self.process.pre_process(self)
```

Comment:
This should be defaulted

Reply:
@JTrentSmart  and @planes  not sure how you want to do this so leaving it for further discussion

Reply:
@planes

Reply:
@planes  completed it